runtime/src/accounts_db.rs

```diff
@@ -4272,6 +4272,52 @@ impl AccountsDb {
         }
     }
 
+    /// Perform the scan for dirty pubkeys in a slot
+    fn do_scan_slot_for_dirty_pubkeys(
+        &self,
+        slot: Slot,
+    ) -> ScanStorageResult<Pubkey, DashSet<Pubkey>> {
+        self.scan_account_storage(
+            slot,
+            |loaded_account: LoadedAccount| {
+                // Cache only has one version per key, don't need to worry about versioning
```

Comment:
stale comment

Reply:
Good catch!

Reply:
nice! yeah we don't care about versioning, just that the pubkey was touched!

Reply:
Done!